openapi-generator-ignore-visit-balance

```diff
@@ -0,0 +1,19 @@
+**/build.gradle
+**/gradlew
+**/docs/
+**/gradle/
+**/gradlew.bat
+**/README.md
+**/settings.gradle
+**/.openapi-generator-ignore
+**/client/infrastructure/
+**/src/test/
+**/api/*
+**/model/*
+!**/BalanceControllerApi.kt
```

Comment:
we only generate the two endpoints and five model classes that we need, excluding all others